roles/matrix-bot-matrix-registration-bot/tasks/setup_install.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+- set_fact:
+    matrix_bot_matrix_registration_bot_requires_restart: false
```

Comment:
This can be removed. Together with the reference to `matrix_bot_matrix_registration_bot_requires_restart` below.

In some other roles, we've used a variable like this for doing data migrations and restarting the service, but we don't really use this for anything here.